tests/unit/extractor/dashboard/tableau/test_tableau_dashboard_extractor.py

```diff
@@ -0,0 +1,75 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+import unittest
+
+from mock import patch
+from pyhocon import ConfigFactory  # noqa: F401
+
+from databuilder import Scoped
+from databuilder.extractor.dashboard.tableau.tableau_dashboard_extractor import TableauDashboardExtractor
+from databuilder.extractor.dashboard.tableau.tableau_dashboard_utils \
+    import TableauDashboardAuth, TableauGraphQLApiExtractor
+
+
+logging.basicConfig(level=logging.INFO)
+
+
+def mock_query(*_args, **_kwargs):
```

Comment:
I'm surprised that mypy isn't mad about there not being a return type on this, I have no idea why it's not... but it doesn't matter here

Reply:
Oh, I know why. There was a mixup along the way and our branch did a rename of  `tests/unit/extractor/dashboard/__init__.py` to `tests/unit/extractor/dashboard/tableau/__init__.py`. So, it's not treated as a module and mypy skips it.
I'll fix.